pubspec.yaml

```diff
@@ -45,3 +45,8 @@ dev_dependencies:
   shelf_test_handler: '^1.0.0'
   test_descriptor: '^1.0.0'
   test_process: '^1.0.0'
+
+dependency_overrides:
```

Comment:
accident?

Reply:
haven't published the new version of stream_channel yet. Will publish it and bump the dep here once we confirm it works.